textpipe/doc.py

```diff
@@ -286,7 +286,10 @@ def _set_default_nlp(self, lang):
         """
         if lang not in self._spacy_nlps:
             self._spacy_nlps[lang] = {}
-        model = spacy.load('{}_core_{}_sm'.format(lang, 'web' if lang == 'en' else 'news'))
+        try:
+            model = spacy.load('{}_core_{}_sm'.format(lang, 'web' if lang == 'en' else 'news'))
+        except IOError:
+            raise MissingModelException(f'Default model for language "{lang}" is not available.')
```

Comment:
Could the be another reason for an `IOError` at this point? Not sure. Might be OK for now.

Reply:
I don't think so: spacy tries to find only this file. I think this should be fine.